projects/observability/src/shared/components/bar-gauge/bar-gauge.component.scss

```diff
@@ -0,0 +1,53 @@
+@import 'mixins';
+
+.bar-gauge {
+  .title {
+    @include overline($gray-7);
+    padding-bottom: 24px;
+  }
+
+  .count {
+    @include subtitle-1($gray-9);
+    padding-bottom: 8px;
+
+    .units {
+      @include body-small($gray-5);
+    }
+  }
+
+  .bar {
+    position: relative;
```

Comment:
Any reason for not using flexbox here?

Reply:
No real reason. This just seemed easier and more straight forward.